src/Console/Command/DescribeCommand.php

```diff
@@ -158,11 +158,10 @@ private function describeRule(OutputInterface $output, $name)
                 $fixer->fix(new StdinFileInfo(), $tokens);
                 $new = $tokens->generateCode();
                 $diff = $differ->diff($old, $new);
-
-                if (null === $codeSample[1]) {
-                    $output->writeln(sprintf(' * Example #%d.', $index + 1));
-                } else {
+                if (isset($codeSample[1])) {
```

Comment:
note to myself:
conflicting with https://github.com/keradus/PHP-CS-Fixer/pull/21